test/unit/specs/dojos/cd-find-dojo.spec.js

```diff
@@ -166,6 +177,12 @@ describe('The Find dojo vue ', () => {
     vm.long = undefined;
     sandbox.stub(vm, 'getDojosByLatLong');
     sandbox.stub(vm, 'getAllDojos');
+    vm.$route = {
```

Comment:
Should be tests for when `q` and `currentLocation` are defined. 3 combinations to be tested.